clients/pkg/promtail/targets/gcplog/push_translation.go

```diff
@@ -73,45 +74,46 @@ func translate(m PushMessage, other model.LabelSet, useIncomingTimestamp bool, r
 		labels[model.LabelName(lbl.Name)] = model.LabelValue(lbl.Value)
 	}
 
-	// add labels coming from scrapeconfig
+	// Add fixed labels coming from the target configuration
 	labels = labels.Merge(other)
 
-	ts := time.Now()
-
 	decodedData, err := base64.StdEncoding.DecodeString(m.Message.Data)
 	if err != nil {
 		return api.Entry{}, fmt.Errorf("failed to decode data: %w", err)
 	}
-	line := string(decodedData)
-
-	if useIncomingTimestamp {
```

Comment:
Q: given we have `useIncomingTimestamp` config is `false` by default. Removing this would break the existing scrape config? No?

https://grafana.com/docs/loki/next/clients/promtail/scraping/#push

Reply:
See [this answer](https://github.com/grafana/loki/pull/7057#discussion_r965006269).